Microsoft.Toolkit.Uwp.SampleApp/SamplePages/GazeInteraction/GazeInteractionPage.xaml.cs

```diff
@@ -55,10 +59,41 @@ public void OnXamlRendered(FrameworkElement control)
             }
 
             gazePointer = GazeInput.GetGazePointer(null);
-
+            ResetGazeCursor();
             CoreWindow.GetForCurrentThread().KeyDown += (CoreWindow sender, KeyEventArgs args) => gazePointer.Click();
         }
 
+        private void ResetGazeCursor()
+        {
+            var size = GazeInput.GetCursorRadius(gazePointer.CursorElement);
+            gazePointer.CursorElement = new Ellipse
```

Comment:
I got an exception right here testing this, when I clicked on the "Open/Close Color Selector" button.
```
System.Exception
  HResult=0x800F1000
  Message=No installed components were detected.

Child collection must not be modified during measure or arrange.
  Source=<Cannot evaluate the exception source>
  StackTrace:
<Cannot evaluate the exception stack trace>
```
Here is the stack:
```
 	[Managed to Native Transition]	
>	Microsoft.Toolkit.Uwp.SampleApp.exe!Microsoft.Toolkit.Uwp.SampleApp.SamplePages.GazeInteractionPage.ResetGazeCursor() Line 69	C#
 	Microsoft.Toolkit.Uwp.SampleApp.exe!Microsoft.Toolkit.Uwp.SampleApp.SamplePages.GazeInteractionPage.SelectedColor.set(Windows.UI.Color value) Line 93	C#
 	Microsoft.Toolkit.Uwp.SampleApp.exe!Microsoft.Toolkit.Uwp.SampleApp.SamplePages.GazeInteractionPage.GazeInteractionPage_obj1_Bindings.UpdateTwoWay_13_Color() Line 161	C#
 	Microsoft.Toolkit.Uwp.SampleApp.exe!Microsoft.Toolkit.Uwp.SampleApp.SamplePages.GazeInteractionPage.GazeInteractionPage_obj1_Bindings.GazeInteractionPage_obj1_BindingsTracking.RegisterTwoWayListener_13.AnonymousMethod__4_0(Windows.UI.Xaml.DependencyObject sender, Windows.UI.Xaml.DependencyProperty prop) Line 203	C#
```